subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/git/GitHubUrlBuilder.java

```diff
@@ -14,17 +14,18 @@
      * Builds GitHub url for the repository
      */
     static String getGitHubUrl(String ghRepo, ShipkitConfiguration conf) {
-        return _getGitHubUrl(conf.getGitHub().getWriteAuthUser(), ghRepo, conf.getLenient().getGitHub().getWriteAuthToken());
+        return _getGitHubUrl(conf.getGitHub().getUrl(), conf.getGitHub().getWriteAuthUser(), ghRepo, conf.getLenient().getGitHub().getWriteAuthToken());
     }
 
     /**
      * Don't use directly, instead call {@link #getGitHubUrl(String, ShipkitConfiguration)}.
      * This way, we use lenient configuration and don't fail when auth token is not provided in shipkit configuration.
      */
     @ExposedForTesting
-    static String _getGitHubUrl(String ghUser, String ghRepo, String writeToken) {
+    static String _getGitHubUrl(String ghUrl, String ghUser, String ghRepo, String writeToken) {
```

Comment:
Wow, what a blunder! Thank you for finding this out! I have big request: can you search the codebase for other usages of "github.com"? I suspect that it is hardcoded in more places? If that's helpful, you can get this PR completed and fix other instances of hardcoded github.com in another PR.

Reply:
Opened issue #595 for this. Couldnt find a test that used the code in question easy, if someone knows of one please point me in the right direction and I can prob. fix this issue soon also.